scripts/50_update_files.sh

```diff
@@ -32,10 +35,39 @@ usage()
     echo "adding them to the scripts runtime environment."
 }
 
+cleanup()
+{
+    # On slow media, umount and/or rmdir can fail with 'resource busy' errors.
+    # To do our best with cleanup, attempt this a few times before giving up.
+    failed=0
+    while test -d "${UPDATE_TARGET}"; do
+        if grep -q "${UPDATE_TARGET}" "/proc/mounts"; then
+            if ! umount "${UPDATE_TARGET}"; then
+                failed=$((failed + 1))
+            fi
+        fi
+
+        if [ -d "${UPDATE_TARGET}" ] && \
+           [ -z "${UPDATE_TARGET##*${NAME_TEMPLATE_UPDATE_TARGET}*}" ]; then
+            if ! rmdir "${UPDATE_TARGET}"; then
+                failed=$((failed + 1))
```

Comment:
quotes

Reply:
are you sure? I omitted them because i remember shell check complaining about that, as it's an integer, not a string. But going to shellcheck.net it doesn't care either way. So i'll add them for consistency. My memory is getting bad!